examples/jsm/capabilities/WebGPU.js

```diff
@@ -26,6 +23,12 @@ class WebGPU {
 
 	}
 
+	static async getStaticAdapter() {
+
+		return await isAvailable;
+
+	}
+
```

Comment:
unnecessary async/await here (isAvailable is already awaited when it's set)

Reply:
although my suggestion would be to keep the async/await here and remove the top level await instead

Reply:
Removed the async/await. To remove the top level await was my initial intention but it's actually necessary to have an accurate result from `navigator.gpu.requestAdapter()`. For instance Puppeeter will use the WebGPU backend as `isAvailable = navigator.gpu !== undefined` is true, while not being able to run WebGPU and so break all Github actions.

Reply:
ok, maybe this is for another PR